manifests/install.pp

```diff
@@ -58,7 +58,7 @@
   case $python::provider {
     pip: {
 
-      package { 'pip':
+      package { 'python-pip':
         ensure  => $pip_ensure,
```

Comment:
would need to set the `name` attribute to be `'pip'` to retain current functionality.